elrond-wasm/src/api/send_api.rs

```diff
@@ -0,0 +1,71 @@
+use super::BigUintApi;
+use crate::types::{Address, ArgBuffer, BoxedBytes, CodeMetadata, TokenIdentifier};
+
+const DIRECT_ESDT_GAS: u64 = 0;
```

Comment:
Shouldn't this be "250,000"? https://docs.elrond.com/developers/esdt-tokens/#transfers

Reply:
Not really, tested on testnet.
It refers to something else - I think you can call a contract with that amount of locked gas. For simple transfers this is not necessary.